### ISSUE TITLE
Player crashes on start

### ISSUE THREAD

**Issue by aurelijusrozenas** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1462]
Hi, 

Im not sure that this is not same problem as in https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/1460, but it did not have enough information.
Player crashes on start displaying error in terminal:
> error: Failed to load bonjour with error: {}

**OS:** Ubuntu 16.04
Player version: 3.3.0
gpmdc.log: https://gist.github.com/aurelijusrozenas/1df7c22c2459328fb85d3036ac0dfbbf

**Comment by MarshallOfSound** [comment]
Can you set the following environment variables

```bash
ELECTRON_ENABLE_STACK_DUMPING=true
ELECTRON_ENABLE_LOGGING=true
```

And run again and paste any stack traces or dumps to a gist and comment here

**Comment by aurelijusrozenas** [comment]
This did not change anything, not more output in terminal or log file. Maybe it would be faster to debug through gitter.im.

**Comment by aurelijusrozenas** [comment]
The problem was solved by updating to 3.4.3 version.
Error message `error: Failed to load bonjour with error: {}` means there is no support chromecast. If you want to use chromecast, you need to install `avahi`.
